dgva-server/src/main/resources/application.properties

```diff
@@ -0,0 +1,19 @@
+# Configure JDBC datasource OR JNDI datasource
+# DGVAPRO Datasource configuration
+spring.datasource.url=@dgvapro.host@
+spring.datasource.username=@dgvapro.user@
+spring.datasource.password=@dgvapro.passwd@
+spring.datasource.driver-class-name=oracle.jdbc.OracleDriver
+spring.datasource.type=oracle.jdbc.pool.OracleDataSource
+spring.jpa.database-platform=org.hibernate.dialect.Oracle10gDialect
+
+#Avoid hibernate ddl schema generation/update or validation
+spring.jpa.hibernate.ddl-auto = none
+#spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.OracleDialect
```

Comment:
I think this is taken care of in the property higher up

`spring.jpa.database-platform=org.hibernate.dialect.Oracle10gDialect`

I don't think this should be needed, this property isn't used in this example https://www.mkyong.com/spring-boot/spring-boot-spring-data-jpa-oracle-example/